demo/demo.html

```diff
@@ -4,29 +4,72 @@
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
     <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
+    <!-- <script src="./temp/sfx-all-plugins-bundle.js"></script> -->
   </head>
+
   <body>
-    <sfx-search-box placeholder="search" clearbutton searchbutton></sfx-search-box>
-    <sfx-sayt visible></sfx-sayt>
-    <sfx-list class="product-grid"></sfx-list>
+    <sfx-search-box id="main-search" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="main-search" visible></sfx-sayt>
+
+    <sfx-search-box id="secondary-search" group="secondary" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="secondary-search" group="secondary" visible></sfx-sayt>
```

Comment:
This is a second searchbox and Sayt for demoing purposes. I can take it out if there is a reason to do so, but we often want to demo against two components. Additionally, end-to-end testing will likely require two components in order to prove that (say) multiple `<sfx-sayt>`s are able to act indepedently.

Reply:
I think this is a good thing to have. We can add a bit of styling later to make it look somewhat saner (so that it doesn't shift past the end of the page, for example).